src/commands/bot/commandsource.js

```diff
@@ -0,0 +1,32 @@
+const { Command, CommandError, SwitchbladeEmbed, GitUtils } = require('../../')
+const { sep } = require('path')
+
+const REPOSITORY_URL = 'https://github.com/SwitchbladeBot/switchblade'
+
+module.exports = class CommandSource extends Command {
+  constructor (client) {
+    super({
+      name: 'commandsource',
+      aliases: ['cmdsource', 'source'],
+      category: 'bot',
+      parameters: [{
+        type: 'command',
+        full: true,
+        required: true,
+        missingError: 'commands:commandsource.missingCommand',
+        acceptHidden: true
+      }]
+    }, client)
+  }
+
+  async run ({ channel, author, t }, command) {
+    const branchOrHash = await GitUtils.getHashOrBranch()
+    if (branchOrHash === null) throw new CommandError(t('commands:commandsource.noRepositoryOrHEAD'))
+    if (!branchOrHash) throw new CommandError(t('commands:commandsource.branchNotUpToDate'))
+
+    channel.send(new SwitchbladeEmbed(author)
+      .setTitle(command.name)
+      .setDescription(`[${t('commands:commandsource.commandSource')}](${REPOSITORY_URL}/blob/${branchOrHash}/${command.path.split(sep).join('/')}) ([${branchOrHash}](${REPOSITORY_URL}/tree/${branchOrHash}))`)
```

Comment:
Maybe try something like this?

![image](https://user-images.githubusercontent.com/25179120/93028231-c8e6ad00-f5e8-11ea-93f2-7802a491ce7a.png)

Reply:
https://github.com/SwitchbladeBot/switchblade/pull/1173/commits/545e5c4c7f356bdde46193c0eafbe65ea46f762c